src/rt/dmain2.d

```diff
@@ -579,6 +587,18 @@ private auto parseExceptionOptions()
     return trap;
 }
 
+private auto parseMemoryErrorOptions()
+{
+    import rt.config : rt_configOption;
+    import core.internal.parseoptions : rt_parseOption;
+    const optName = "memoryError";
+    auto option = rt_configOption(optName);
```

Comment:
`const`?

Reply:
Can't pass that to `rt_parseOption` then.